valohai_yaml/schema/endpoint.yaml

```diff
@@ -64,6 +68,13 @@ properties:
             type: number
           max:
             type: number
+      devices:
+        type: object
+        description: "Devices required e.g. 'nvidia.com/gpu: 1' for one NVIDIA GPU."
+        patternProperties:
+          "^.+$":
+            type: number
+            minimum: 1
```

Comment:
GPUs would always `integer`s but maybe there are other device types that use floats 🤷